app/views/form/page.html.erb

```diff
@@ -30,6 +30,9 @@
       <%= form_with model: @case_log, url: form_case_log_path(@case_log), method: "post" do |f| %>
         <%= f.govuk_error_summary %>
         <% @page.questions.map do |question| %>
+          <% if question.read_only? %>
+            <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--m">
```

Comment:
So I think this question/field only shows if you have JS enabled. Would we still want the divider to show if the field wasn't?

Reply:
That's a good question, probably not, I'll fix it